src/contracts/interfaces/IEigenPod.sol

```diff
@@ -79,29 +83,37 @@ interface IEigenPod {
     /// @notice an indicator of whether or not the podOwner has ever "fully restaked" by successfully calling `verifyCorrectWithdrawalCredentials`.
     function hasRestaked() external view returns (bool);
 
-    /// @notice block number of the most recent withdrawal
-    function mostRecentWithdrawalBlockNumber() external view returns (uint64);
+    /// @notice block timestamp of the most recent withdrawal
+    function mostRecentWithdrawalTimestamp() external view returns (uint64);
+
+    /// @notice Returns the validatorInfo struct for the provided pubkeyHash
+    function validatorPubkeyHashToInfo(bytes32 validatorPubkeyHash) external view returns (ValidatorInfo memory);
 
 
     ///@notice mapping that tracks proven partial withdrawals
-    function provenPartialWithdrawal(uint40 validatorIndex, uint64 slot) external view returns (bool);
+    function provenWithdrawal(bytes32 validatorPubkeyHash, uint64 slot) external view returns (bool);
```

Comment:
in-line documentation is now inaccurate, as this is used for both partial and full withdrawals

Reply:
fixed